lib/game.rb

```diff
@@ -6,32 +6,53 @@
 require_relative "./game_checker"
 
 class Game
-  attr_reader :display, :board, :player_1
-  def initialize(display, presenter, board, players)
-    @display = display
-    @presenter = presenter
-    @board = board
-    @player_1 = players[0]
-    @player_2 = players[1]
+  attr_reader :display, :board, :player_1, :player_2, :active_player, :game_checker
+  def initialize(config_object)
+    @display = config_object[:display]
+    @presenter = config_object[:presenter]
+    @board = config_object[:board]
+    @player_1 = config_object[:player_1]
+    @player_2 = config_object[:player_2]
+    @game_checker = config_object[:game_checker]
 
     @active_player = @player_1
   end
 
   def start
     @display.output(WELCOME)
+    print_board
   end
 
   def play
-    print_board
-    @board.place_mark(@player_1.mark, @player_1.get_input(@display))
-    print_board
-    @board.place_mark(@player_2.mark, @player_2.get_input(@display))
-    print_board
+    loop do
+      @board.place_mark(@active_player.mark, @active_player.get_input(@display))
+      print_board
+      switch_player
+      if @game_checker.game_over?(@board, @player_1, @player_2)
+        @display.output(GAME_OVER)
+        break
```

Comment:
Do you need @display.output(GAME_OVER) if print_board is displaying the outcome?  Also, if print_board is displaying the outcome, the loop can be
```
until @game_checker.game_over?
end
```